src/OrbitGl/TimeGraph.cpp

```diff
@@ -518,7 +518,9 @@ double TimeGraph::GetUsFromTick(uint64_t time) const {
 
 uint64_t TimeGraph::GetTickFromWorld(float world_x) const {
   double ratio =
-      world_width_ != 0 ? static_cast<double>((world_x - world_start_x_) / world_width_) : 0;
+      world_width_ != 0
```

Comment:
There is an issue if world_width - right_margin is equal to 0. For not being error prone I proposed:
double visible_width = world_width - right_margin_;
and using visible_width instead.

- And I'm also not sure what to do in the case of visible_width < 0. What do you think? Should we check this? Or compare visible_width > 0 intead of != 0?

Reply:
Done! Thanks for finding this!